index.html

```diff
@@ -38,146 +38,138 @@
+          <p class="pa2 bb b--black dwyl-dark-gray pb3 tl-ns montserrat f5 mt2 pb4-l">More than 250,000 artefacts, made accessible
+            and searchable!</p>
+          <img src="./img/common/science-museum-logo.jpg" alt="Science Museum logo" class="w-50 w-80-m center db pb3"/>
         </a>
-      </div>
-      <div class="w-10-ns pt2 ph2 pb3 dib-ns dtc-ns v-top tc center">
-        <a href="/portfolio"  class="link dwyl-dark-gray">
-          <div class="pa2 bg-white br2 h5 shadow-2">
-            <p class="pa2 bb b--black pb3 tl-ns montserrat f6">Automating data collection means our TA's
-              can get on with what really matters.</p>
-            <img src="./img/common/inclusive-classrooms-logo.png" alt="Inclusive Classrooms logo" class="w-80 w5-l pt3 pt3-l"/>
-          </div>
+        <a href="/portfolio" class="link db dtc-m dtc-l bg-white pa2 br2 relative shadow-6 mb4 mb0-ns">
+          <p class="pa2 bb b--black dwyl-dark-gray pb3 tl-ns montserrat f5 mt2">Automating data collection means our TA's
+            can get on with what really matters.</p>
+          <img src="./img/common/inclusive-classrooms-logo.png" alt="Inclusive Classrooms logo" class="w-60 w-80-m pt2 pt2-l pb3 center db"/>
         </a>
-      </div>
-      <div class="w-10-ns pt2 ph2 pb3 dib-ns dtc-ns v-top tc center">
-        <a href="/portfolio" class="link dwyl-dark-gray">
-          <div class="pa2 bg-white br2 h5 shadow-2">
-            <p class="pa2 bb b--black pb3 tl-ns montserrat f6">We increased community engagement with an
-            exclusive social network.</p>
-            <img src="./img/common/ellen-macarthur-foundation-logo.png" alt="Ellen MacArthur Foundation logo" class="w-90 w5-l pt4 pt4-l"/>
-          </div>
+        <a href="/portfolio" class="link db dtc-m dtc-l bg-white pa2 br2 relative shadow-6">
+          <p class="pa2 bb b--black dwyl-dark-gray pb3 tl-ns montserrat f5 mt2">We increased community engagement with an
+          exclusive social network.</p>
+          <img src="./img/common/ellen-macarthur-foundation-logo.png" alt="Ellen MacArthur Foundation logo" class="w-70 w-90-m pt3 pt3-l pb3 center db"/>
         </a>
       </div>
       <div class="tc">
-        <a href="#contact" class="dib link mv3 pa2 dwyl-bg-mint white br2 f5 w-40-ns w-80">Start Your Own Success Story</a>
+        <a href="#contact" class="dib link mt5 mb4 pt3 pb3 ph4 dwyl-bg-mint white br2 f4 w-auto">Start Your Own Success Story</a>
       </div>
     </section>
     </section>
 
-    <section class="dwyl-bg-mint ph4 pt5 pb4 tc">
-      <img src="./img/common/mvp.png" alt="We can be/make your MVP" class="center tc pt4"/>
+    <section class="ph4 pt5 pb4 tc">
+      <h2 class="pa0 ma0 f1 b dwyl-dark-gray text-shadow-1">We can <span class="relative transparent"><img src="/img/common/be-make.png" class="dwyl-mint be-make dib relative"/>Make</span> your MVP*</h2>
```

Comment:
I'm assuming I'm getting this because I'm just opening the page in the browser and it doesn't know where to load the image from?
<img width="749" alt="screen shot 2017-10-09 at 19 03 42" src="https://user-images.githubusercontent.com/4185328/31351945-e8729036-ad24-11e7-9957-dd9eaddf2376.png">

How does this play out in screen readers or SEO impact?

Reply:
The weird span/image mix is just leftover mess from trying to get #308 working. Now I know this restyle is going ahead I'll make an image for the section.